packages/vue-client/src/components/boards/MulticolorGraphBoard.vue

```diff
@@ -21,7 +21,9 @@ const props = defineProps<{
   board_config: BoardConfig;
 }>();
 
-const intersections = createBoard(props.board_config, Intersection);
+const intersections = computed(() =>
```

Comment:
Putting the intersections into a computed caused performance problems for me. For the game page it should be enough to compute the intersections once.
For the variant demo I noticed that this causes a problem, but it would be great if we can find another solution that doesn't slow the game page down.

Reply:
`computed` (or some sort of reactivity) seems like an appropriate thing to do here.  What kind of performance issues were you seeing?

> For the game page I think it would be enough to compute the intersections once.

I'm not exactly sure how Vue router works, but if we ever support navigating to different games from the games page, this may lead to stale data.

Reply:
I remember noticing a lag between clicking (for playing a move), and the stone appearing. But I just tried to reproduce, and didn't have such problems. So now I'm thinking this is probably fine.

But I think its worth noting that: Whenever a move is made, the server sends all game information to the clients. The config hasn't changed (but the client doesn't know that) and so the intersections are computed anew. I believe this also causes the board to be rerendered, since it depends on the intersections.